### PR TITLE
Add Azure DevOps Server docs page

### LINKED ISSUE #85: Docs for Azure DevOps Server Integrations
## User story.

Users have instructions for git integrations but we are missing Azure DevOps Server: https://meta.stoplight.io/docs/platform/2.-workspaces/configure-git/a.configuring-git.md

## Describe the solution you'd like

A similar set of instructions for Azure DevOps Server.

### PR DESCRIPTION
**Pull request by heitortsergent**
Fixes #85.

- [ ] Are there any steps required from an Azure DevOps Server configuration side of things?
- [ ] What are the minimum scopes necessary for the personal access token?
  - I did play around with this a little bit, and even if I tried selecting all scopes manually (not the All Scopes option), I still get an error message when trying to connect an account...

### CODE REVIEW

File: 2.-workspaces/configure-git/i.azure-devops-server.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+## Azure DevOps Server Setup
+
+There are two things we'll need from Azure:
+
+1. The URL for your instance of Azure DevOps Server. The format should be: `https://{your_server_ip:port}/tfs/{your_collection_name}`.
+2. A **personal access token**. To generate a personal access token:
+    1. In the upper-right corner of the page, select your profile photo, then select **Security** or **Personal access tokens** depending on your version of Azure DevOps Server.
+    2. Select **Add** or **New Token**.
+    3. Add a name in the **Description** field, set the expiration time, and select **All Scopes** for the **Authorized Scopes** field.
```
Comment (chohmann) [inline 2.-workspaces/configure-git/i.azure-devops-server.md:19]: I faintly remember needing to do the "all scopes" option, though I'm not 100% confident in that. @rainum do you remember if it's necessary to select the "All Scopes" option for your PAT for ADS? It sounds like @heitortsergent tried selecting all the different scopes individually, but still ran into issues until he selected the "All Scopes" option.

**Review by chohmann — APPROVED** [review]
This look good to me! I've asked @rainum for some confirmation on needing "All Scopes" for your PAT.

File: 2.-workspaces/configure-git/i.azure-devops-server.md (hunk 1 above)
Comment (rainum) [reply 2.-workspaces/configure-git/i.azure-devops-server.md:19]: Yeah, I do recall this issue but I wasn't able to find any mentions in the slack/github issues. Would be great to find some additional info about this issue somewhere in the Microsoft support forums, so we can include it in our docs.

Comment (heitortsergent) [reply 2.-workspaces/configure-git/i.azure-devops-server.md:19]: If you're both ok with this, maybe I can merge this as is for now just so it's available for users, and I can open another issue to try and track down why we need `All Scopes` and update this doc later on?

Comment (chohmann) [reply 2.-workspaces/configure-git/i.azure-devops-server.md:19]: works for me!

Comment (rainum) [reply 2.-workspaces/configure-git/i.azure-devops-server.md:19]: Sounds good!

**Review by pamgoodrich — APPROVED** [review]

**Pull request merged**